std/bitmanip.d

```diff
@@ -2722,7 +2669,6 @@ if (isFloatOrDouble!T)
                          char, wchar, dchar/*,
                          float, double*/))
     {{
-        scope(failure) writefln("Failed type: %s", T.stringof);
```

Comment:
On a second thought I'm not so sure about removing these `scope(failure)` blocks. This looks like it might be useful to debug failures on DAutoTest given that D's `assert` is quite dumb.
What do you or other people think? Should we leave this in and do the following at the top / within the test?

```d
version(StdUnittest) import std.stdio : writefln;
```

Reply:
I'm not a fan of code bases like DMD that leave in lots of residual `print`s that add clutter but might one day be useful to debug the code. Better to either use a debugger, add in the `writeln`s yourself when you need them, or fix `assert` to give better diagnostics.

Reply:
Feel free to leave those in on a case basis. First thing I'd do if I had to debug that would be to insert such stuff. I don't think that versioned import is a good idea.